graphql_compiler/compiler/ir_lowering_sql/__init__.py

```diff
@@ -1,5 +1,30 @@
 # Copyright 2018-present Kensho Technologies, LLC.
-
+"""
+Lower GraphQL compiler IR to a SQL Tree representation.
+
+Two goals are accomplished in this lowering pass:
+1. All filters and outputs are associated with a unique query path, later allowing these filters and
```

Comment:
Mind making the numbering consistent with the references below? `(1), (2)` etc. This will also (accidentally) make the second lines below indented 4 spaces, which will prevent editors from being confused about the proper indentation depth.